src/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Cell Tower Coverage</title>
+
+    <!-- META TAGS -->
+    <!-- required for IE and bootstrap -->
+    <meta http-equiv="X-UA-Compatible" content="IE=9" />
+
+    <meta charset='utf-8'>
+    <link rel="shortcut icon" href="http://mapserv.utah.gov/cdn/images/favicon.ico"/>
+    <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
+    <meta name="description" content="A basic web site highlighting the Utah automated geographic reference center's map cache's, widgets, and web services." />
+    <meta name="keywords" content="utah, map, map of utah, utah data, gis, utah gis, gis data, sgid, utah gis portal, agrc, atlas, utah atlas, online utah atlas" />
+
+    <!-- CSS -->
+    <link rel='stylesheet' href='app/resources/App.css'>
+</head>
+<body class='claro'>
+    <div data-dojo-type='app/App'></div>
+
+    <!-- build:template
+    <script data-dojo-config="deps:['app/packages', 'app/run']" src='dojo/dojo.js'></script>
+    /build -->
+
+    <!-- build:remove -->
+    <script data-dojo-config="isDebug: 1" src='dojo/dojo.js'></script>
+    <script src='app/packages.js'></script>
+    <script src='app/run.js'></script>
+    <!-- /build -->
+
+    <!-- GOOGLE ANALYTICS -->
+    <script>
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+        ga('create', 'UA-11849964-7', 'utah.gov');
```

Comment:
This key needs to be updated. I can help get a new one when you know what the production URL will be. I'm guessing something like `mapserv.utah.gov/celltowercoverage`?